src/pages/docs/step-ca/policies.mdx

```diff
@@ -0,0 +1,295 @@
+email       |@example.com               |jdoe@example.com, janedoe@example.com      |jdoe@www.example.com, jdoe@somehost.com
+
+
+
+### URIs
+
+Rules for evaluating URIs are evaluated as follows:
+
+- URI domains are normalized into their [internationalized representation (RFC 5890)](https://datatracker.ietf.org/doc/html/rfc5890).
+- URI rules must contain an authority part and cannot contain IP addresses, in correspondence with [RFC 5280](https://datatracker.ietf.org/doc/html/rfc5280#section-4.2.1.10).
+- URI schemes are not matched. When configuring an URI rule, the scheme is ignored.
+- Matching URI domains is performed by matching URI domain labels, similar as what's done for DNS domains.
+
+Wildcards in URI rules are evaluated similar to wildcards for domain names.
+URIs cannot contain literal wildcards, unlike DNS domains.
+
+#### Examples
+
+Type        | Rule                      | Matches                                           | Non-Matches
+------------|:-:                        |:-:                                                |:-:
+uri         |https://host.example.com   |https://host.example.com,sftps://host.example.com  |https://www.example.com
+uri         |https://*.example.com      |https://host.example.com,https://www.example.com   |https://example.com, https://somehost.com
+uri         |*.example.com              |https://host.example.com,https://www.example.com   |https://example.com, https://somehost.com
+
+
+
+### Subject Common Name
+
+The Subject Common Name is a bit of a special case.
+When no rules are configured specifically for the Common Name, the Common Name in an X.509 certificate will be evaluated against the rules and evaluation logic for the other name types, according to what type it is.
+For example, if the Common Name has a value of `www.example.com`, it will be evaluated against the rules for DNS domains.
+
+Type    | Rule              | Matches           | Non-Matches
+--------|:-:                |:-:                |:-:
+dns     |*.local            |ca.local           |ca.example.com
+ip      |192.168.0.0/24     |192.168.0.1        |10.0.0.1
+email   |@local             |ca@local           |ca@example.com
+uri     |https://*.local    |https://ca.local   |https://ca.example.com
+
+Not every Common Name is a valid DNS domain, IP, email address or URI, however, so that's why it's also possible to provide rules specifically for Common Names.
+Rules for Common Names are evaluated using a strict string comparison.
+
+Type    | Rule              | Matches                               | Non-Matches
+--------|:-:                |:-:                                    |:-:
+cn      |Custom CA Name     |Custom CA Name                         |Different CA Name
+
+
+## SSH Policies
+
+Policies for SSH certificates consist of rules for user and host certificates.
+A policy for SSH user certificates can have rules for the following:
+
+- Email address domains
```

Comment:
Nice. this used to be called `allowedDomains` on the provisioner? Are we keeping that config parameter around?

Reply:
I see now that provisioner-level policies are not allowed in open source.

Reply:
I think you're referring to the parameter on the OIDC provisioner? There's an internal note about that, but there it's called `domains` instead. They're different things, restricting the credential itself vs. what names are requested, so that configuration is kept.

The parameter is on a different nesting level and managed differently, so I don't know if it's going to confuse things.